docs/MapColonies/vector/Services/geocoding/README.md

```diff
@@ -0,0 +1,1024 @@
+            "properties": {
+                "OBJECT_COMMAND_NAME": "111",
+                "TIED_TO": "olimpiade",
+                "ENTITY_HEB": "control point",
+                "LAYER_NAME": "CONTROL_GIL_GDB.CTR_CONTROL_POINT_CROSS_N",
+                "TYPE": "ITEM",
+                "matches": [ { "layer": "CONTROL_GIL_GDB.CTR_CONTROL_POINT_CROSS_N", "source": "control_gil_v5_test", "source_id": [] } ],
+                "names": { "default": [ "111" ], "display": "Route olimpiade Control Point 111" },
+                "score": 1.89712
+            }
+        }
+    ]
+}
+```
+</details>
+
+Items are simply buildings\ items in the field. Item's name consists 4-digit number. It might be reused in different tiles! <br/>
+
+<details>
+    <summary>JSON Schema of Item Response</summary>
+
+```json
+{
+    "type": "FeatureCollection",
+    "geocoding": {
+        "version": "0.1.0",
+        "query": {
+            "command_name": "1234",
+            "limit": 5,
+            "disable_fuzziness": true
+        },
+        "response": {
+            "results_count": 1,
+            "max_score": 2.664375,
+            "match_latency_ms": 5
+        }
+    },
+    "features": [
+        {
+            "type": "Feature",
+            "geometry": {
+                "coordinates": [ [
+                        [ 12.432792582620323, 41.9327692706986 ],
+                        [ 12.432648028923637, 41.93209008126263 ],
+                        [ 12.43295235249525, 41.93189198298137 ],
+                        [ 12.435105441764364, 41.93235609798671 ],
+                        [ 12.435516278586334, 41.93274663116725 ],
+                        [ 12.43637599267521, 41.93308056343986 ],
+                        [ 12.436026020567567, 41.93386161853829 ],
+                        [ 12.432792582620323, 41.9327692706986 ] ] ],
+                "type": "Polygon"
+            },
+            "properties": {
+                "OBJECT_COMMAND_NAME": "1234",
+                "TILE_NAME": "RIT",
+                "SUB_TILE_ID": "37",
+                "LAYER_NAME": "CONTROL.ITEMS",
+                "ENTITY_HEB": "hotel",
+                "TYPE": "ITEM",
+                "matches": [ { "layer": "CONTROL.ITEMS", "source": "control_gil_v5_test", "source_id": [] } ],
+                "names": { "default": [ "1234" ], "display": "Item 1234" },
+                "score": 2.664375
+            }
+        }
+    ]
+}
+```
+</details>
+
+## Location Name Based Search: 
+:::tip
+Check [`Location Search request examples`](#location-search-examples).
+:::
+
+We have created a location search engine. In the query parameter simple search a place that you want to find. For example, if you search for `White House, Washington DC` you will get matching results for your query. <br/>
+You can also search for supported regions and sources so you can filter the results for a specific region/ specific source (it will result results from all regions and sources by default). <br/>
```

Comment:
```suggestion
You can also search for supported regions and sources in order to filter the results for a specific region or source (by default it will return all regions and sources). <br/>
```